### DISCUSSION TITLE
Gestione altri tipi di presenza

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#429] [Idee e proposte]
Buonasera, volevo sapere, in linea di massima, per quando è previsto l'inserimento della gestione di diverse tipologie di presenza (richiesta https://github.com/iisgiua/giuaschool/discussions/403#discussion-4495911)

**Comment by trinko** [comment]
Non sono in grado di dirlo. Al momento sto seguendo altre priorità.
